Form/Extension/DependencyInjectionExtension.php

```diff
@@ -69,15 +64,40 @@ class DependencyInjectionExtension implements FormExtensionInterface
     private $guesserLoaded = false;
 
     /**
+     * @var array
+     */
+    private $mappingExtensionTypes;
+
+    /**
+     * @var array|bool
+     */
+    private $reverseMappingTypes;
+
+    /**
+     * DependencyInjectionExtension constructor.
+     *
      * @param ContainerInterface $container
      * @param array              $typeServiceIds
      * @param array              $typeExtensionServiceIds
      * @param array              $guesserServiceIds
      * @param array              $mappingTypes
      * @param array              $extensionTypes
      */
-    public function __construct(ContainerInterface $container, array $typeServiceIds, array $typeExtensionServiceIds, array $guesserServiceIds, array $mappingTypes = array(), array $extensionTypes = array())
-    {
+    public function __construct(
+        ContainerInterface $container,
+        array $typeServiceIds,
+        $typeExtensionServiceIds,
+        $guesserServiceIds,
+        array $mappingTypes = [],
+        array $extensionTypes = []
+    ) {
```

Comment:
I love your style, the way Symfony does this is just plain stupid IMO. Sadly, other @sonata-project/contributors do not agree with me so I'm going to ask you to put that back on one line :(